plugins/hypervisors/kvm/test/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtUtilitiesHelperTest.java

```diff
@@ -37,8 +38,8 @@ public void testSSHKeyPaths() {
          * Hardcoded paths are not what we want in the longer run
          */
         assertEquals("/root/.ssh", helper.retrieveSshKeysPath());
-        assertEquals("/root/.ssh/id_rsa.pub.cloud", helper.retrieveSshPubKeyPath());
-        assertEquals("/root/.ssh/id_rsa.cloud", helper.retrieveSshPrvKeyPath());
+        assertEquals("/root/.ssh" + File.separator + "id_rsa.pub.cloud", helper.retrieveSshPubKeyPath());
```

Comment:
@DaanHoogland True, but the test still works :)

@borisroman Yes, but this is just to make the Unit Tests work under Windows.